src/OpenApi/Model/RequestBody.php

```diff
@@ -17,16 +17,16 @@ final class RequestBody
 {
     use ExtensionTrait;
 
-    public function __construct(private string $description = '', private ?\ArrayObject $content = null, private bool $required = false)
+    public function __construct(private ?string $description = null, private ?\ArrayObject $content = null, private bool $required = false)
```

Comment:
funny enough `content` is Required in the spec: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#request-body-object

Reply:
The PR's originated where I wanted to edit the description of an endpoint, but by doing so, I needed to rewrite the whole `content`.

The PR code make the `Model\RequestBody` itself differs from `OpenAPI` own specification.
But the whole `OpenApiFactory` will bring back the compliance while the specification by overwriting the `content` if it's set to `null`.